apps/nova/src/constants/user.ts

```diff
@@ -12,3 +15,36 @@ export enum TutorOnboardingStep {
    */
   Profile = 3,
 }
+
+export const governorates: Record<IUser.City, LocalId> = {
+  "1": "global.governorates.1",
```

Comment:
We can do even better. 

- Use the enum value. 
- Don't use the city index in the local id name. 

```suggestion
  [IUser.City.Cairo]: "global.governorates.cairo",
```